ignite/contrib/handlers/param_scheduler.py

```diff
@@ -468,15 +468,32 @@ def __init__(self, schedulers, durations, save_history=False):
         self.schedulers = schedulers
         self.durations = durations
 
-        self.optimizer = self.schedulers[0].optimizer
-        if not (all(id(s.optimizer) == id(self.optimizer) for s in self.schedulers)):
+        param_schedulers = [s for s in self.schedulers if isinstance(s, ParamScheduler)]
+        param_group_schedulers = [s for s in self.schedulers if isinstance(s, ParamGroupScheduler)]
+
+        param_optimizers = [s.optimizer for s in param_schedulers]
+        param_group_optimizers = [s.optimizer for sc in param_group_schedulers for s in sc.schedulers]
```

Comment:
I was thinking if it could make sense to provide `ParamGroupScheduler.optimizer` attribute as a list of all optimizers used inside `ParamGroupScheduler.schedulers` ? 
and thus simplify this code too

Reply:
Don't know if it simplify the code because we have to flat the list of list (of optimizers). What do you think ?

Reply:
I was thinking about that `ParamGroupScheduler.optimizer` is a property that returns `[s.optimizer for s in ParamGroupScheduler.schedulers]` such that all the code L471-L475 could be done:
```python
param_optimizers = [s.optimizer for s in self.schedulers]
```

Reply:
It can be so simple. The list will contain optimizer objects (from ParamScheduler) and list of optimizer objects (from ParamGroupScheduler). Thus we have to flat an heterogenous list. @vfdev-5 ok with that ?

Reply:
Let's try that